src/components/ha-camera-stream.ts

```diff
@@ -37,6 +40,19 @@ class HaCameraStream extends LitElement {
 
   private _hlsPolyfillInstance?: Hls;
 
+  private _useExoPlayer = false; 
+
+  private async _getUseExoPlayer(): Promise<boolean> {
+    if (this.hass!.auth.external) {
```

Comment:
Return early when we don't need to do anything.
```suggestion
    if (!this.hass!.auth.external) {
      return false;
    }
```